addons/website_livechat/tests/test_livechat_basic_flow.py

```diff
@@ -255,6 +255,7 @@ def test_channel_visitor_data(self):
                 "res.country": [
                     {"code": "BE", "id": self.env["ir.model.data"]._xmlid_to_res_id("base.be")}
                 ],
+                "res.lang": [{"id": self.visitor.lang_id.id, "name": self.visitor.lang_id.name}],
```

Comment:
We should know which lang id and name we are expecting and explicit them, tests lose value if they re-do what the code itself already does.